yt_dlp/utils/traversal.py

```diff
@@ -228,6 +228,12 @@ def apply_path(start_obj, path, test_type):
             if not casesense and isinstance(key, str):
                 key = key.casefold()
 
+            if key in (any, all):
+                has_branched = False
+                filtered_objs = (obj for obj in objs if obj not in (None, {}))
+                objs = (next(filtered_objs, None) if key is any else list(filtered_objs),)
```

Comment:
```suggestion
                objs = (next(filtered_objs, None) if key is any else list(filtered_objs), )
```
readablity

Reply:
Not having a space after the comma is uncommon. Default is `(1,)`

Reply:
It's also what [PEP8 recommends](<https://peps.python.org/pep-0008/#pet-peeves>):
> Avoid extraneous whitespace in the following situations:
> - Between a trailing comma and a following close parenthesis:
> ```py
> # Correct:
> foo = (0,)
> ```
> ```py
> # Wrong:
> bar = (0, )
> ```

Reply:
For long expressions like this, no space it makes it hard to recognize as a tuple imo

Reply:
I think the best would be to use an intermediate variable and have `objs = (result,)` or split it into an if statement:
```suggestion
                if key is any:
                    objs = (next(filtered_objs, None),)
                else:
                    objs = (list(filtered_objs),)
```

Reply:
just put a space imao. Afaik, Most linters don't even have a rule for this, and we certainly use both variations elsewhere

Reply:
If we use the formatter it will get reformatted eventually. I think the multiline statements reads even easier anyways

Reply:
Personally, I liked the single line version, but 🤷